types/jsforce/index.d.ts

```diff
@@ -1,5 +1,5 @@
 // Type definitions for jsforce 1.9
-// Project: https://github.com/jsforce/jsforce
```

Comment:
Same for this.

Why change https to http?

Reply:
https://github.com/jsforce/jsforce/pull/860